### ISSUE TITLE
User Groups

### ISSUE THREAD

**Issue by rgine** [TheHive-Project/TheHive#1183]
### Request Type
Feature Request

TheHive should add support for user's to be added to "groups", and "groups" should be a selectable assignee. 

Security teams might have multiple groups that could get assignments in TheHive. For example "SOC Team", "IR Team", "Forensic Team", "SOC Management", etc. Users could be a member of one or more groups. 

The "SOC Team" might want to add an assignment for "Forensic Analysis" to the "Forensic Team".

**Comment by veeral-patel** [comment]
Got it, so basically you’d like to be able to assign cases to groups?

**Comment by rgine** [comment]
Correct, and each user should be able to be a part of one or more groups. Instead of having to assign a case to a specific person, it should be able to be assigned to a group (made up of one or multiple users).

**Comment by otgrkiss** [comment]
+1

**Comment by veeral-patel** [comment]
@rgine @otgrkiss Got it. Right now, a user can see all the tickets that are assigned to him.

Would you propose we add another page where the user can see all the tickets that are assigned to each of the group he/she's in?

**Comment by pyperanger** [comment]
+1

**Comment by bl00droot** [comment]
+1

**Comment by rmonetto** [comment]
+1

**Comment by fabbiomcosta** [comment]
+1
